### PR TITLE
add kuzmin test for anisotropy constants

### LINKED ISSUE #96: Implement a test in kuzmin that checks wheather Ku and K1 work as expected
Previous comment from Martin: We should have one test that checks that passing Ku works as expected, not sure if we have one at the moment.

### PR DESCRIPTION
**Pull request by AdamsMP92**
## Summary

Add a regression test for `kuzmin_properties` to ensure that both `K1` and `Ku` are accepted as zero-temperature anisotropy input.

The test checks that passing `me.K1(...)` and `me.Ku(...)` as `K1_0` produces equivalent temperature-dependent anisotropy results.

closes #96

### CODE REVIEW

File: changes/110.misc.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Added a regression test ensuring `kuzmin_properties` accepts both `K1` and `Ku` as zero-temperature anisotropy inputs.
```
Comment (andrea-petrocchi) [inline changes/110.misc.md]: I would not create a changelog for adding tests. I guess the users would not see that anyway.

File: tests/test_kuzmin.py [hunk 2]
```diff
@@ -218,6 +218,35 @@ def test_kuzmin_properties_no_K1_0():
...
+        Ms=Ms_data,
+        T=T_data,
+        Tc=Tc,
+        Ms_0=Ms_0,
+        K1_0=me.Ku(1e5, unit=u.J / u.m**3),
+    )
+
+    assert isinstance(result_K1.K1, _K1_function_of_temperature)
+    assert isinstance(result_Ku.K1, _K1_function_of_temperature)
+    assert result_K1.K1(0) == result_Ku.K1(0)
```
Comment (andrea-petrocchi) [inline tests/test_kuzmin.py]: 0 is included in T_data so this line could be deleted.

Comment (andrea-petrocchi) [reply tests/test_kuzmin.py]: What you could check instead is that
```python
result_K1.K1(0) == me.K1(1e5, unit=u.J / u.m**3)
result_Ku.K1(0) == me.K1(1e5, unit=u.J / u.m**3)
```

**Review by andrea-petrocchi — CHANGES_REQUESTED** [review]
It looks good overall. I have left some comments to be considered.

File: changes/110.misc.md (hunk 1 above)
Comment (AdamsMP92) [reply changes/110.misc.md]: Deleted.

File: tests/test_kuzmin.py (hunk 2 above)
Comment (AdamsMP92) [reply tests/test_kuzmin.py]: True. Changed to suggestions.

**Review by andrea-petrocchi — APPROVED** [review]
